src/templates/book-review/book-review.js

```diff
@@ -0,0 +1,64 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+import parse from 'html-react-parser';
+import './book-review.scss';
+/**
+ * Slideshow component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The slide content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function BookReview({ content }) {
+  const {
+    authorText,
+    media: {
+      authorImage: { url: authorUrl },
+      bookImage: { url: bookUrl }
+    },
+    bookText
+  } = content;
+  const authorImage = { backgroundImage: `url("${authorUrl}")` };
+  const bookImage = { backgroundImage: `url("${bookUrl}")` };
+
+  return (
+    <>
+      <div className="dokk1-book-review">
+        <div className="text-area">
+          <p>{parse(bookText)}</p>
```

Comment:
The html should be sanitized before it is inserted. Maybe https://www.npmjs.com/package/dompurify